sandbox/sounds/html/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
This file may have been copied and included by mistake in the sounds folder as well